yaml_files/simphony_metadata.yml

```diff
@@ -503,32 +496,21 @@ CUDS_KEYS:
 
   INLET_OUTLET:
     definition: Inlet outlet boundary condition (outlet condition is zero gradient and inlet given variable value)
-    parent: CUBA.CONDITION
+    parent: CUBA.VALUE_CONDITION
     models: [CUBA.CONTINUUM]
-    CUBA.VARIABLE:
-      shape: (:)
-      default: []
-    CUBA.MATERIAL:
-      shape: (:)
-      default: []
 
   WETTING_ANGLE:
-    parent: CUBA.CONDITION
+    parent: CUBA.VALUE_CONDITION
     definition: Volume fraction wall boundary condition with specified contact angle
     models: [CUBA.CONTINUUM]
-    CUBA.VARIABLE:
-      shape: (:)
-      default: []
-    CUBA.CONTACT_ANGLE:
-      default: 45.0
+    variable: CUBA.VOLUME_FRACTION
```

Comment:
> variable is equivalent to CUBA.VARIABLE.

It is not. not according to the current specs. Otherwise, we need to add `models` as well as CUBA.MODELS, and since we are there, also parent. 

Besides, using the case to define this is very deceiving and unheard of. The point is that if you want a variable property to be readonly, we should have a way of doing so.